lib/stsci/tools/capable.py

```diff
@@ -10,6 +10,7 @@
 
 import os, sys
 PY3K = sys.version_info[0] > 2
+descrip = "basic capabilities file, last edited 29 July 2016"
```

Comment:
Where is this used? I don't find any explanation for this PR and the title is very vague.